spot-client/src/spot-remote/ui/loaders/withRemoteControl.js

```diff
@@ -84,7 +92,14 @@ export class RemoteControlLoader extends AbstractLoader {
      * @inheritdoc
      */
     componentWillUnmount() {
-        remoteControlService.removeEventListener(this._onServiceEvent);
+        remoteControlService.removeListener(
+            SERVICE_UPDATES.DISCONNECT,
```

Comment:
At this point yes, that's the only time there's a disconnect that isn't recoverable. The change happened when reconnect logic was moved into RCS itself.

Reply:
If yes then maybe the event's name should be something which contains "unathorized" in the name rather than generic disconnect, or a reason passed to the disconnect

Reply:
or maybe use "_onDisconnect" for the handler method's name like done in the other place below